public/js/actions/subscriptions.js

```diff
@@ -33,43 +33,68 @@ export function getUserSubscriptions(jquery=$) {
 }
 
 
-export function createSubscription(braintreeToken, productId,
-                                   creditCard, jquery=$,
+export function createSubscription(productId, payMethod,
+                                   braintreeToken, jquery=$,
                                    BraintreeClient=braintree.api.Client) {
```

Comment:
I think this is going to make the calling code too confusing. I suggest we move it to parameters. Luckily, we can use defaults, like this:
````
createSubscription({productId=undefined, payMethod=undefined, creditCard=undefined, 
                    jquery=$, BraintreeClient=braintree.api.Client}) {
}
````

this way you don't have to overload and introspect `payMethod`. It would be more explicit.